textattack/augmentation/augmenter.py

```diff
@@ -137,3 +137,6 @@ def augment_text_with_ids(self, text_list, id_list, show_progress=True):
             all_text_list.extend([text] + augmented_texts)
             all_id_list.extend([_id] * (1 + len(augmented_texts)))
         return all_text_list, all_id_list
+
+    def __repr__(self):
+        return type(self).__name__
```

Comment:
can you copy the `Attack` repr to print `self.transformation` and `self.constraints` please